lib/transform/transformers.js

```diff
@@ -30,7 +30,10 @@ export function assets (asset) {
   transformedAsset.fields.file = reduce(
     asset.fields.file,
      (newFile, file, locale) => {
-       newFile[locale] = omit(file, 'url', 'details')
+       if (newFile[locale].uploadFrom) {
+         delete newFile[locale].uploadFrom
+       }
```

Comment:
in second thought yeah we can keep it there and if it fails it will be catched [here](https://github.com/contentful/contentful-batch-libs/blob/master/lib/push/assets.js#L8) in case of the file was already deleted from the upload endpoint